interface/src/interfaces/controllers/basic-left-right/BasicLeftRight.vue

```diff
@@ -0,0 +1,12 @@
+<template src="./basic-left-right.html"></template>
+
+<script>
+export default {
+  name: 'BasicLeftRightInterface',
+  emits: ['arrow-left-down', 'arrow-left-up', 'arrow-right-down', 'arrow-right-up']
```

Comment:
tutaj i w innych wystąpieniach tych stringów znowu jest ta sama sytuacja: to magic stringi. Chcemy tego uniknąć. Jako, że występuje również w pliku .html to proponuję przepisać treść pliku html bezpośrednio do tej templaty, a plik usunać